static/js/scroll_bar.js

```diff
@@ -60,7 +60,7 @@ exports.initialize = function () {
             "<style> @media (max-width: 1165px) { .compose-content, .header-main .column-middle { margin-right: " +
                 (7 + sbWidth) +
                 "px !important; } } " +
-                "@media (max-width: 775px) { .fixed-app .column-middle { margin-left: " +
+                "@media (max-width: 767px) { .fixed-app .column-middle { margin-left: " +
```

Comment:
This is from addfcbfc30bbcddbac4137649469332f063baad4 -- we should probably do some sort of test in a browser like Chrome on Windows (that has a fixed-width scrollbar) in that width range.

Reply:
Tested! Works fine. This commit now present as https://github.com/zulip/zulip/commits/114d38613b0c82279faa7c6aabbd2ba4bf064ac8